### PR TITLE
/home and /spawn enhancement

### PR DESCRIPTION
**Pull request by jimchen5209**
Rewrite version of #298
## Changes
- /home to spawn point of vanilla (bed, respawn anchor) with spawn point check (E.g. bed or respawn anchor is missing or occupied, respawn anchor out of charge)
- /spawn to world spawn with safe check (overworld), teleport to the world's spawn point which is not at overworld.

### CODE REVIEW

File: src/main/kotlin/one/oktw/galaxy/command/commands/Home.kt [hunk 1]
```diff
@@ -46,21 +49,81 @@ class Home : Command {
...
+        if (!spawnPoint.isPresent) {
+            player.sendMessage(TranslatableText("block.minecraft.spawn.not_valid").styled { it.withColor(Formatting.RED) }, false)
+            lock -= player.uuid
         } else {
-            lock += player.uuid
             GlobalScope.launch {
+                // Add back charge in countdown stage
+                if (world != null) {
+                    val blockState = world.getBlockState(spawnPointPosition)
+                    if (blockState.block is RespawnAnchorBlock) {
```
[the remark this answers is not in the document]
Comment (jimchen5209) [reply src/main/kotlin/one/oktw/galaxy/command/commands/Home.kt:79]: get 要在 serverThread 喔

**Review by bill96012 — APPROVED** [review]
Test passed

File: src/main/resources/accessor.mixin.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "required": true,
+  "package": "one.oktw.galaxy.mixin.accessor",
+  "compatibilityLevel": "JAVA_14",
```
Comment (james58899) [inline src/main/resources/accessor.mixin.json:4]: 14?

Comment (jimchen5209) [reply src/main/resources/accessor.mixin.json:4]: Master 的其他 mixin 好像也是 14 🤔

**Review by james58899 — APPROVED** [review]

**Pull request merged**